app/controllers/api/v1/participants_controller.rb

```diff
@@ -6,21 +6,9 @@ module Api
   module V1
     class ParticipantsController < Api::ApiController
       include ApiTokenAuthenticatable
-      include ApiPagination
-      include ApiCsv
-      include ApiFilter
 
       def index
-        respond_to do |format|
-          format.json do
-            participant_hash = ParticipantSerializer.new(paginate(participants)).serializable_hash
-            render json: participant_hash.to_json
-          end
-          format.csv do
-            participant_hash = ParticipantSerializer.new(participants).serializable_hash
-            render body: to_csv(participant_hash)
-          end
-        end
+        redirect_to controller: "ecf_participants", action: "index", params: request.params and return
```

Comment:
I can't see any entry point to this. Why do we need to redirect here?

Reply:
Oh, I forgot to delete it from earlier attempts 😂 Good spot!